File: spaceship_crm/scala/spaceship_crm_scala_play_frm/app/frm/entity/Activity.scala
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package frm.entity

import java.time.LocalDateTime

import play.api.libs.json.Json

import scala.beans.BeanProperty

case class Activity (
  @BeanProperty id: Long,
  @BeanProperty name: String,
  @BeanProperty startDate: LocalDateTime,
  @BeanProperty endDate: LocalDateTime
)

object Activity {
  implicit val userFormat = Json.format[Activity]
}

File: spaceship_crm/java8/spaceship_crm_java8_rest_jooq/src/main/java/com/spaceship/crm/entity/Activity.java
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package com.spaceship.crm.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Activity {

    private Long id;

    private String name;

    private Date startDate;
    private Date endDate;

}
